src/localization_plugin.h

```diff
@@ -97,6 +111,14 @@ class LocalizationPlugin : public ed::Plugin
 
     bool laser_offset_initialized_;
 
+    // random pose generation
+
+    geo::Vec2 min_map_, max_map_;
+    unsigned long last_map_size_revision_;
+
+    geo::Transform2 generateRandomPose(std::function<void()> update_map_size);
```

Comment:
I think that you could get a better overview of this class by grouping the functions and the member variables